packages/store/tests/build-action-handler/fixtures/add-country-action-handler.ts

```diff
@@ -0,0 +1,13 @@
+import { inject } from '@angular/core';
+import { ActionDirector } from '@ngxs/store';
+
+import { COUNTRIES_STATE_TOKEN } from './state';
+import { AddCountry } from './actions';
+
+export const addCountryActionHandler = () => {
+  const actionDirector = inject(ActionDirector);
```

Comment:
Just a thought regarding your implementation of this lazy loaded action handler...
You could pass the reference to the `ActionDirector` as an argument to this function. Then it would remove the need to run it in an injection context.
Up to you whether you change it or leave it. Doesn't matter to me.
```suggestion
export const addCountryActionHandler = (actionDirector: ActionDirector) => {
```